modules/new_profile/php/NDB_Form_new_profile.class.inc

```diff
@@ -34,14 +34,9 @@ class NDB_Form_New_Profile extends NDB_Form
         $user =& User::singleton();
 
         $site_arr = $user->getData('CenterIDs');
-        foreach ($site_arr as $key=>$val) {
-            $site[$key]        = & Site::singleton($val);
-            $isStudySite[$key] = $site[$key]->isStudySite();
-            $isDCCSite[$key]   = $site[$key]->getCenterName();
-        }
-        $oneIsStudySite = in_array("1", $isStudySite);
-        $oneIsDCCSite   = in_array("DCC", $isDCCSite);
-        if ($oneIsStudySite or $oneIsDCCSite) {
+        $userInDCCSite   = in_array("1", $site_arr);
```

Comment:
Here I check for CenterID rather than the hardcoded "DCC" which is not the CenterName anyway on any new Loris install (it is Data Coordinating center not DCC as DCC is the Alias).